Sources/Basics/Observability.swift

```diff
@@ -54,19 +54,23 @@ public class ObservabilitySystem {
             description: "top scope",
             parent: .none,
             metadata: .none,
-            diagnosticsHandler: factory.diagnosticsHandler
+            diagnosticsHandler: factory.diagnosticsHandler,
+            metricsHandler: factory.metricsHandler
         )
     }
 
-    private struct NOOPFactory: ObservabilityFactory, DiagnosticsHandler {
+    private struct NOOPFactory: ObservabilityFactory, DiagnosticsHandler, MetricsHandler {
         var diagnosticsHandler: DiagnosticsHandler  { self }
+        var metricsHandler: MetricsHandler { self }
 
         func handleDiagnostic(scope: ObservabilityScope, diagnostic: Diagnostic) {}
+        func handleTimer(scope: ObservabilityScope, label: String, duration: DispatchTimeInterval) {}
```

Comment:
Adding the start time in addition to the duration (or, really, recording any two of: start time, duration, or end time) would let us plot a timeline of what happened and when, which would be really great.  Could the start time of he activity be added here?  I realize that this is not consistent across hosts, but on a single host as we're talking about here, just the wall clock time would be tremendously useful.